get-started/index.md

```diff
@@ -1,67 +1,54 @@
-
-The following samples are recommended for getting started:
-
- * [Step by Step Sample](/samples/step-by-step/): Illustrates essential NServiceBus concepts by showing how to build a simple system.
- * [Endpoint configuration choices](/samples/endpoint-configuration/): Walks through the most common choices required when creating a first endpoint. It will also show the configuration APIs needed to implement those choices.
- * [On Premise Show Case](/samples/show-case/on-premise/): An implementation of a fictional store that shows many features of NServiceBus working together.
 
+### Sending a message
 
-## Downloading
+```cs
+await endpoint.Send(new SubmitOrder { OrderId = orderId });
+```
 
-NServiceBus consists of a many components (some optional) which are deployed through [NuGet Packages](https://www.nuget.org). The most important elements, required in most scenarios, are the [transport](/transports/) and [persistence](/persistence/). The transport is an abstraction over low-level messaging infrastructure (e.g. MSMQ or SQL Server), the persistence provides support for some NServiceBus features like [delayed-delivery](/nservicebus/messaging/delayed-delivery.md), [publish-subscribe](/nservicebus/messaging/publish-subscribe/) and [sagas](/nservicebus/sagas/).
+### Processing a message
 
-The majority of the NuGet packages are listed under the [NServiceBus NuGet User](https://www.nuget.org/profiles/nservicebus) with the main library is deployed via the [NServiceBus NuGet package](https://www.nuget.org/packages/NServiceBus/).
+```cs
+public class OrdersHandler : IHandleMessages<SubmitOrder>
+{
+    public async Task Handle(SubmitOrder message, IMessageContext context)
+    {
+        Console.WriteLine($"Order received {message.OrderId}");
 
+        await context.Publish(new OrderAccepted { OrderId = message.OrderId });
+    }
+}
 ```
-PM> Install-Package NServiceBus
-```
 
-There are also many of extensions and utilities that build on the Platform in a variety of ways. These are listed under [Extensions](/components/).
 
-The other parts of the Platform ([ServiceControl](/servicecontrol/), [ServiceInsight](/serviceinsight/) and [ServicePulse](/servicepulse/)) are deployed through either the [Platform Installer](/platform/installer/) (mainly for development environment) or via a [Direct Download](https://particular.net/downloads) of an installer (mainly for production and integration environments).
+## Other resources
+
+* [High-level content](/get-started/high-level-content.md) - Recommended articles to gain an understanding of the key concepts and technologies used within the Particular Service Platform.
+* [Tutorials](/tutorials/) - Step-by-step guides to help master NServiceBus and the Particular Service Platform
+* [Samples](/samples/) - Learn by exploring code in fully functional solutions that can be run from Visual Studio. Go to the [introduction to samples](/samples/) or [skip to the list of samples](/samples/#related-samples) 
```

Comment:
introduction to samples still points to /samples.

Reply:
Same as old line 35 did (the "introduction" is the content at the top)